src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java

```diff
@@ -304,7 +276,7 @@ public AboutToStartOrSubmitCallbackResponse addUploadOrder(
     ) throws Exception {
         CaseData caseData = CaseUtils.getCaseData(callbackRequest.getCaseDetails(),objectMapper);
         Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
-        log.info("/manage-orders/add-upload-order before caseData ===> " + callbackRequest.getCaseDetails());
+        log.info("/manage-orders/add-upload-order before caseData ===> " + callbackRequest);
```

Comment:
remove logger. follow the same approach as for other. Speak to Sonali if needed.